### PR TITLE
Decimal to Hexadecimal Conversion in C

### PR DESCRIPTION
**Pull request by MayureeS**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

Given a number in Decimal form, convert it into its Hexadecimal form.
Ex:
Input : 10
Output: A

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4623

### CODE REVIEW

**Comment by MayureeS** [comment]
@HarshCasper @ankitaggarwal23 Kindly review my PR.

File: C/math/Decimal_to_Hex.c [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	{
+		printf("%c" , hex[i]);
+	}
+	return 0 ;
+}
+/*
+Sample Example :
+I/O : 45
+O/P : 2D
+Explanation: 45 when converted into hexadecimal is equal to 2D. So it outputs 2D.
```
Comment (ankitaggarwal23) [inline C/math/Decimal_to_Hex.c]: Remove this and Write the outputs in the way your code is producing.

**Review by ankitaggarwal23 — COMMENTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by MayureeS** [comment]
@ankitaggarwal23 I have made the said changes.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**